.github/CODEOWNERS

```diff
@@ -950,7 +950,7 @@ packages/kbn-plugin-helpers @elastic/kibana-operations
 packages/kbn-react-field @elastic/kibana-app-services
 packages/kbn-repo-source-classifier @elastic/kibana-operations
 packages/kbn-repo-source-classifier-cli @elastic/kibana-operations
-packages/kbn-rule-data-utils @elastic/apm-ui
+packages/kbn-rule-data-utils @elastic/security-detections-response @elastic/actionable-observability @elastic/response-ops
```

Comment:
I'd be curious to learn why the CI added this. :thinking:

Reply:
I am too... I will revert this change as it's not an intentional change on my part.

Reply:
I wouldn't be surprised if the automation adds it right back :wink:

Reply:
I changed this fwiw, see https://github.com/elastic/kibana/pull/144608